content/en/cloudcraft/getting-started/crafting-better-diagrams.md

```diff
@@ -0,0 +1,64 @@
+---
+title: "Crafting Better Diagrams: Cloudcraft's Live Diagramming and Filtering"
+kind: documentation
+---
+
+## Overview
+
+Introduction.
+
+## Prerequisites
+
+Before you begin, you must connect your AWS or Azure account to Cloudcraft. For more information, see:
+
+- [Connect your AWS account with Cloudcraft][1]
+- [Connect your Azure account with Cloudcraft][2]
+
+## Enable new live experience
+
+To enable the new live experience, toggle the **New Live Experience** switch at the top of the **Live** tab in Cloudcraft.
+
+{{< img src="cloudcraft/getting-started/crafting-better-diagrams/enable-new-experience.png" alt="Screenshot highlighting the switch to enable the new live experience beta feature in the Cloudcraft interface with a red arrow pointing to the switch." responsive="true" style="width:100%;">}}
+
+If you're a new user, you may already have the new live experience enabled by default.
+
+## Select account and region
+
+Click the dropdown under the **Account** section and select the account you want to scan. If you only added one AWS account to Cloudcraft, it's automatically selected for you.
+
+{{< img src="cloudcraft/getting-started/crafting-better-diagrams/select-account.png" alt="Screenshot highlighting the dropdown to select an account to want to scan in the Cloudcraft interface with a red arrow pointing to the dropdown." responsive="true" style="width:100%;">}}
+
+Under **Region**, select the regions you want to scan. By default, `Global` and `US East (N. Virginia)` are selected, but you can click the **More** button to select or search for additional regions.
```

Comment:
We select `Global` and whichever region they configured as their default region by default, with `US East (N. Virginia)` being the fallback region if they don't have any default region configured. Similarly, Azure regions selected by default are `Global` and their configured default region, or `East US`